### PR TITLE
Bug 1810642:  Snapshot feature for is part of 4.5v

### PR DESCRIPTION
**Pull request by a2batic**
Moves snapshot feature to 4.5 for OCS.

### CODE REVIEW

**Comment by afreen23** [comment]
On small PRs, one gets picky. 
> Moves snapshot feature to 4.5 for OCP.

This is for OCS 4.5

**Comment by a2batic** [comment]
/retitle Bug 1810642:  Snapshot feature for is part of 4.5v

File: frontend/packages/ceph-storage-plugin/src/features.ts [hunk 1]
```diff
@@ -19,19 +19,19 @@ export const detectIndependentMode: ActionFeatureFlagDetector = (dispatch) =>
...
   );
 
-export const OCS_VERSION_4_4_FLAG = 'OCS_VERSION_4_4';
+export const OCS_VERSION_4_5_FLAG = 'OCS_VERSION_4_5';
 
-export const isOCS44Version = (subscription: K8sResourceKind): boolean => {
+export const isOCS45Version = (subscription: K8sResourceKind): boolean => {
   const version = subscription?.status?.currentCSV;
-  return version && version.includes('ocs-operator.v4.4');
+  return version && version.includes('ocs-operator.v4.5');
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/features.ts]: Something like this maybe:
`version.split('.')[2] >= 5`

File: frontend/packages/ceph-storage-plugin/src/features.ts [hunk 2]
```diff
@@ -19,19 +19,19 @@ export const detectIndependentMode: ActionFeatureFlagDetector = (dispatch) =>
     },
   );
 
-export const OCS_VERSION_4_4_FLAG = 'OCS_VERSION_4_4';
+export const OCS_VERSION_4_5_FLAG = 'OCS_VERSION_4_5';
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/features.ts]:
```suggestion
export const OCS_4_5_AND_ABOVE_FLAG = 'OCS_4_5_AND_ABOVE';
```

File: frontend/packages/ceph-storage-plugin/src/features.ts [hunk 3]
```diff
@@ -20,19 +20,20 @@ export const detectIndependentMode: ActionFeatureFlagDetector = (dispatch) =>
     },
   );
 
-export const OCS_VERSION_4_4_FLAG = 'OCS_VERSION_4_4';
+export const OCS_VERSION_4_5_AND_ABOVE_FLAG = 'OCS_VERSION_4_5_AND_ABOVE_FLAG';
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/features.ts]: If you look at the flag name the value does not have flag in it.

Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/features.ts]: We are detecting 4_5. No need for the above part.

Comment (a2batic) [reply frontend/packages/ceph-storage-plugin/src/features.ts]: ack

**Pull request merged**